ddtrace/internal/writer/writer.py

```diff
@@ -454,7 +458,7 @@ def __init__(
         max_payload_size=None,  # type: Optional[int]
         timeout=None,  # type: Optional[float]
         dogstatsd: Optional[DogStatsd] = None,
-        report_metrics=False,  # type: bool
+        report_metrics=True,  # type: bool
```

Comment:
intended? 🤔

Reply:
Yes, this parameter wasn't actually used before